test/errors/scope-resolution/use-shadows-formal2.1.good

```diff
@@ -2,6 +2,8 @@ use-shadows-formal2.chpl:1: In module 'M0':
 use-shadows-formal2.chpl:2: error: 'b' has multiple definitions
 use-shadows-formal2.chpl:3: note: redefined here
 use-shadows-formal2.chpl:14: In function 'foo':
-use-shadows-formal2.chpl:15: warning: module-level symbol is hiding function argument 'a'
-use-shadows-formal2.chpl:7: note: and then through the 'use' statement here
-use-shadows-formal2.chpl:2: note: found 'b' defined here
+use-shadows-formal2.chpl:16: warning: potentially surprising shadowing for 'a'
+use-shadows-formal2.chpl:15: note: it refers to a symbol found through the 'use' of 'M1' here
+use-shadows-formal2.chpl:7: note: and then through the 'use' of 'M0' here
+use-shadows-formal2.chpl:2: note: leading to 'b' defined here
```

Comment:
My feeling is that these lines are extraneous in "brief" mode. You get a whole wall of text, some of which may take you into library code, and it obfuscates the error a bit.

Reply:
Eh, Lydia called it out as better on a very similar test in https://github.com/chapel-lang/chapel/pull/23511#discussion_r1341835911 . I thought our policy was that brief mode should print all of the same locations, just potentially with less explanatory text and without code snippets.

Reply:
I like this better than the previous version because if you look at it without looking at the code (which would be common if you made a whole bunch of changes between compiles), it's confusing why 'b' is even being mentioned when you were looking for 'a'.  I initially thought there was a bug with our shortened dyno error messages where it was missing part of an error or something.

Whereas in this version, it's more clear that it's definitely connected to the previous line (and the line before that, and the line before that).